tests/linux/test_backend_utils.cpp

```diff
@@ -115,6 +156,36 @@ void test_image_resizing(const char* img, const mp::MemorySize& img_virtual_size
 
     EXPECT_EQ(process_count, expected_final_process_count);
 }
+
+template <class Matcher>
+void test_image_conversion(const char* img_path, const char* expected_img_path, const char* qemuimg_info_output,
+                           const mp::ProcessState& qemuimg_info_result, bool attempt_convert,
+                           const mp::ProcessState& qemuimg_convert_result, mp::optional<Matcher> throw_msg_matcher)
+{
+    auto process_count = 0;
+    auto mock_factory_scope = mpt::MockProcessFactory::Inject();
+    const auto expected_final_process_count = attempt_convert ? 2 : 1;
+
+    mock_factory_scope->register_callback([&](mpt::MockProcess* process) {
+        ASSERT_LE(++process_count, expected_final_process_count);
+        if (process_count == 1)
+        {
+            auto msg = QByteArray{qemuimg_info_output};
+
+            simulate_qemuimg_info_with_json(process, img_path, qemuimg_info_result, msg);
+        }
+        else
+            simulate_qemuimg_convert(process, img_path, expected_img_path, qemuimg_convert_result);
```

Comment:
```suggestion
        else
        {
            simulate_qemuimg_convert(process, img_path, expected_img_path, qemuimg_convert_result);
        }
```

Reply:
I see another place that has the braces missing in the `else` block when the `if` block has braces.  I'll just fix all of those up in one commit.